lms/tasks/rsa_key.py

```diff
@@ -0,0 +1,19 @@
+from datetime import datetime, timedelta
+
+from lms.services import RSAKeyService
+from lms.tasks.celery import app
+
+
+@app.task
+def rotate_keys():
```

Comment:
Docstring here would be good explaining what this does without having to travel and read around.